apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/dashboard/_components/Layout.tsx

```diff
@@ -4,18 +4,96 @@ import { EvaluationDto } from '@latitude-data/core/browser'
 import {
   BlankSlateStep,
   BlankSlateWithSteps,
+  Button,
   Icon,
   TableBlankSlate,
+  Text,
   useCurrentCommit,
   useCurrentDocument,
   useCurrentProject,
 } from '@latitude-data/web-ui'
+import { connectEvaluationsAction } from '$/actions/evaluations/connect'
+import useLatitudeAction from '$/hooks/useLatitudeAction'
+import { useNavigate } from '$/hooks/useNavigate'
 import { ROUTES } from '$/services/routes'
 import useEvaluations from '$/stores/evaluations'
+import useSuggestedEvaluations, {
+  SuggestedEvaluation,
+} from '$/stores/suggestedEvaluations'
 import Link from 'next/link'
 
 import BatchEvaluationsTable from './BatchEvaluationsTable'
 
+function SuggestedEvaluations() {
+  const document = useCurrentDocument()
+  const { project } = useCurrentProject()
+  const { commit } = useCurrentCommit()
+  const { data: suggestions, isLoading } = useSuggestedEvaluations(
+    document.content,
+  )
+  const navigate = useNavigate()
+  const { mutate } = useEvaluations()
+  const { execute, isPending } = useLatitudeAction(connectEvaluationsAction)
+  const onConnect = async (suggestion: SuggestedEvaluation) => {
+    const [data] = await execute({
+      projectId: project.id,
+      templateIds: [suggestion.id],
+      evaluationUuids: [],
+      documentUuid: document.documentUuid,
+    })
+
+    if (data) {
+      mutate()
+      const connectedEvaluation = data[0]!
+      navigate.push(
+        ROUTES.projects
+          .detail({ id: project.id })
+          .commits.detail({ uuid: commit.uuid })
+          .documents.detail({ uuid: document.documentUuid })
+          .evaluations.detail(connectedEvaluation.evaluationId).root,
+      )
+    }
+  }
+
+  if (isLoading) return null
+  if (!suggestions.length) return null
```

Comment:
If the placeholder just pops in out of nothing, it moves and rearranges the rest of steps, which can be annoying. Could we use a placeholder instead?

Reply:
I opted not to add a placeholder because
1 - the ai might suggest no templates so in that case we don't show anything
2 - with the animations i've added it feels quite good actually

Reply:
done!